.github/workflows/ci.yaml

```diff
@@ -11,20 +11,6 @@ on:
 jobs:
   tox:
     runs-on: ubuntu-latest
-    services:
```

Comment:
Can you give me a quick high level explanation of what the changes in the CI are?

Reply:
the service (elasticsearch) was being loaded for the tox jobs but not for the unit tests (where is required)

Reply:
Also, I couldn't get to load elasticsearch on macOS (because it lacks docker) so I have separated the ubuntu and macOS unit test

Reply:
I see! That makes sense. IMO we should not overcomplicate things. As far as I remember, the ES tests are going to be skipped when no ES server is detected. 

Let's discuss this in person:)